src/Demo/Server.hs

```diff
@@ -17,7 +17,7 @@ import           Network                     hiding (connectTo)
 main = do
   state <- createState (PortNumber 9878)
   flip (runStateT . runRicochet) state $ do
-    versions %= (M.insert 1 (\_ -> return ()))
+    versions %= M.insert 1 closeConnection
```

Comment:
The intention here was not to close the connection but to do nothing and execute the rest of the `do` block instead.